tests/end_to_end/test_client_json.py

```diff
@@ -15,13 +15,17 @@
 import ga4gh.cli as cli
 import ga4gh.datarepo as datarepo
 import tests.utils as utils
+from freezegun import freeze_time
```

Comment:
We have a convention for imports, so we know where modules come from at a glance. From STYLE.md

```
Imports should be structured into the following groups:

1. Any ```__future__``` imports
2. Any standard library imports
3. Any third party library imports
4. Any package level imports
```

So, datetime should go with json and unittest above and freezegun should have it's own group immediately below this.